mmv1/products/cloudrun/Service.yaml

```diff
@@ -687,6 +691,78 @@ properties:
                                 The name of the service to place in the gRPC HealthCheckRequest
                                 (see https://github.com/grpc/grpc/blob/master/doc/health-checking.md).
                                 If this is not specified, the default behavior is defined by gRPC.
+                    - name: 'readinessProbe'
+                      type: NestedObject
+                      description: |-
+                        Periodic probe of container readiness.
+                      properties:
+                        - name: 'timeoutSeconds'
+                          type: Integer
+                          description: |-
+                            Number of seconds after which the probe times out.
+                            Defaults to 1 second. Must be smaller than period_seconds.
+                          default_value: 1
+                        - name: 'periodSeconds'
+                          type: Integer
+                          description: |-
+                            How often (in seconds) to perform the probe.
+                            Default to 10 seconds.
+                          default_value: 10
```

Comment:
I'm seeing the default values are included in the API reference, we should utilize `default_from_api` instead of client-side default values.

- https://cloud.google.com/run/docs/reference/rpc/google.cloud.run.v1#google.cloud.run.v1.Probe